warehouse/upload.go

```diff
@@ -1515,16 +1515,17 @@ func (job *UploadJobT) setStagingFilesStatus(stagingFiles []*StagingFileT, statu
 	return
 }
 
-func (job *UploadJobT) hasLoadFiles(tableName string) (bool, error) {
+func (job *UploadJobT) hasLoadFiles(tableName string) (exists bool, err error) {
```

Comment:
Can we make it one query instead of querying every time per table.

Reply:
Made this change. Now during the start of exporting data stage. We are populating the loadFIlesTableMap and we are using it throughout.